[05:41:26] {main.py:24} INFO - ___________NEW RUN STARTING___________
[05:41:26] {main.py:25} INFO - Code started at: 2022-02-12 05:41:26.924104
[05:41:26] {/opt/virtualenvs/python3/lib/python3.8/site-packages/urllib3/connectionpool.py:971} DEBUG - Starting new HTTPS connection (1): dumbot.drdum.repl.co:443
[05:41:27] {/opt/virtualenvs/python3/lib/python3.8/site-packages/urllib3/connectionpool.py:452} DEBUG - https://dumbot.drdum.repl.co:443 "GET / HTTP/1.1" 200 17
[05:41:27] {/home/runner/DumBot/keep_alive.py:27} INFO - Status code: 200
[05:41:27] {/opt/virtualenvs/python3/lib/python3.8/site-packages/urllib3/util/retry.py:333} DEBUG - Converted retries value: 3 -> Retry(total=3, connect=None, read=None, redirect=None, status=None)
[05:41:27] {/opt/virtualenvs/python3/lib/python3.8/site-packages/google/auth/transport/requests.py:177} DEBUG - Making request: POST https://oauth2.googleapis.com/token
[05:41:27] {/opt/virtualenvs/python3/lib/python3.8/site-packages/urllib3/connectionpool.py:971} DEBUG - Starting new HTTPS connection (1): oauth2.googleapis.com:443
[05:41:27] {/opt/virtualenvs/python3/lib/python3.8/site-packages/urllib3/connectionpool.py:452} DEBUG - https://oauth2.googleapis.com:443 "POST /token HTTP/1.1" 200 None
[05:41:27] {/opt/virtualenvs/python3/lib/python3.8/site-packages/urllib3/connectionpool.py:971} DEBUG - Starting new HTTPS connection (1): www.googleapis.com:443
